command-line-local-shortener/templates/index.html

```diff
@@ -5,25 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Golang URL shortener</title>
-    <link rel="stylesheet" href="style.css" />
+    <!-- <link rel="stylesheet" href="style.css" />  -->
+
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
   </head>
 
   <body>
     <!-- <p>example paragraph<strong>example</strong> example
         <strong>example</strong>example</p>  -->
-    <!-- maybe do an onclick instead of making a whole new form??  -->
+
     <form action="http://localhost:8080/create" method="POST">
-      <h1>URL shortener</h1>
+      <h1 class="d-flex justify-content-center">URL shortener</h1>
 
-      <fieldset>
-        <div id="short">
+      <fieldset class="container">
+        <div class="mb-3">
           <label for="fname">Custom ID:</label> <br />
```

Comment:
The `for` attribute of `label` should correspond to the `id` of the form control it is labeling.

See:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for